_pkgdown.yml

```diff
@@ -45,6 +45,9 @@ reference:
       - getPopulation
       - getPopulations
       - deletePopulationAuto
+      - updateGate

```

Comment:
This one will get picked up by `ends_with("Gate")` on line 41.

Just realized this is sort of a hodgepodge ... not alphabetical, not grouped by gate/populations, not grouped by CRUD function. 🤦‍♂ Fine for now, can reorg later.